tests/install.sh

```diff
@@ -6,7 +6,7 @@
 # from some of the commands in this file (#39).
 set -xe
 
-SERVER='http://10.77.77.1:4000/directory'
+SERVER='http://10.77.77.1:4001/directory'
```

Comment:
What was the reason for changing the port?

Reply:
Boulder's port `4000` is for HTTP ACME v1. Port `4001` is HTTP ACME v2.